### ISSUE TITLE
Resolutions above 1080p cause the game to reset to defaults

### ISSUE THREAD

**Issue by eggmanstudio** [MadDeCoDeR/Classic-RBDOOM-3-BFG#66]
DoomBFA seems to dislike resolutions above 1080p. If I try to set a fullscreen resolution that's higher than 1920x1080 (on either axis), the game defaults to 720p at 60 fps. Notably, the original RBDoom3BFG did not have this issue.

3 monitors, RTX 2070, Ryzen 7 3700X, 32gb DDR4, Windows 10.

**Comment by MadDeCoDeR** [comment]
Is the resolution set from the in-game menu or with launch arguments?

**Comment by eggmanstudio** [comment]
> 
> 
> Is the resolution set from the in-game menu or with launch arguments?

In-game.

**Comment by MadDeCoDeR** [comment]
Does it switch the monitor when you change the resolution?

**Comment by eggmanstudio** [comment]
> Does it switch the monitor when you change the resolution?

No, it gives me separate options for each monitor (My main monitor is Monitor 2). 

In any case I found a workaround. I put an autoexec.cfg in the base folder and used set r_customWidth and set r_customHeight for 1440p.

**Comment by MadDeCoDeR** [comment]
Does it throw any errors on the Console (+set com_allowConsole 1) or the Logs (+set logfile 1, C:/Users/<username>/Saved Games/Id Software/DOOM BFA/base/qconsole.log)?

**Comment by eggmanstudio** [comment]
> 
> 
> Does it throw any errors on the Console (+set com_allowConsole 1) or the Logs (+set logfile 1, C:/Users/{username}/Saved Games/Id Software/DOOM BFA/base/qconsole.log)?

I removed my autoexec and turned on logging, then opened the game and tried to change the resolution. This is the log it spat out:
[qconsole.log](https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/files/6657083/qconsole.log)

I then tried loading the game again and changing the resolution to 2560x1440 on Monitor 2 and this was the resulting qconsole file:

[qconsole.log](https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/files/6657087/qconsole.log)

I don't know what most of it means so feel free to peruse.

**Comment by MadDeCoDeR** [comment]
As it seems (to me at least) for some reason your PC reports that the resolution you are trying to set is not supported and therefor it fails and then it goes to one of my fallbacks which revert the resolution and the FPS to the minimum required.
